console/src/main/webapp/manager/app/assets/index.html

```diff
@@ -14,7 +14,8 @@
 
 <body ng-app="manager" ng-strict-di ng-controller="AppController">
 
-  <iframe src="/header/?active=console" scrolling="no" frameborder="0"></iframe>
+  <geor-header style="width:100%;height:90px" active-app="console"></geor-header>
+  <script src="http://localhost:5500/header.js"></script>
```

Comment:
There's still a forgotten dev value here. However, why is this value used and not the ones above in the `header.jsp`?

Reply:
Fixed :)